Source/TCPError.swift

```diff
@@ -30,17 +30,17 @@ public enum TCPError: ErrorProtocol {
     case operationTimedOut(description: String, data: Data)
     case closedSocket(description: String)
 
-    static func lastReceiveErrorWithData(source: Data, bytesProcessed: Int) -> TCPError {
+    static func lastReceiveError(withData source: Data, bytesProcessed: Int) -> TCPError {
```

Comment:
yeah, maybe data is't so helpful when you got error. i don't think it can help with receive error, and in case of send error you can log data one layer above in try/catch, right?